.gitlab/source_test/linux.yml

```diff
@@ -51,10 +48,11 @@
 .upload_junit_source:
   - $CI_PROJECT_DIR/tools/ci/junit_upload.sh
 
-.linux_tests_with_upload:
-  extends: .linux_tests
-  after_script:
-    - !reference [.upload_junit_source]
+.upload_coverage:
+  # Upload coverage files to Codecov. Never fail on coverage upload.
+  - source /root/.bashrc
```

Comment:
I'd avoid putting the `source/root/.bashrc` in the `.upload_coverage` function. To me this line is there just because of the way our runner works and is not really related with the `upload_coverage`. It would also prevent you from using it on Windows runner I think

Reply:
Discussed offline: the `source/root/.bashrc` is already run in the `aws_ssm_get_wrapper.sh` script + the `after_script` doesn't have the env var defined, giving me an `aws command not found` if I don't add it.